spec/requests/review_pages_spec.rb

```diff
@@ -230,6 +230,33 @@
        it "shows the edit page" do
         current_path.should == edit_review_path(review)
        end
+
+       it "updates the review date and feedback deadline" do
+         fill_in "review_review_date", with: "2020-06-07"
+         fill_in "review_feedback_deadline", with: "2020-01-21"
+
+         click_button "Save Changes"
+
+         current_path.should == review_path(review)
+         review.reload
+         review.review_date.should == Date.new(2020, 6, 7)
+         review.feedback_deadline.should == Date.new(2020, 01, 21)
+         page.should have_selector('.flash-success',
+          text: 'Review was updated. Your coach and invitees have been notified of the new date.')
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.